src/uwtools/drivers/forecast.py

```diff
@@ -147,10 +149,12 @@ def output(self):
         Create list of SRW output files and stage them in the working directory.
         """
 
-    def job_card(self):
+    def job_card(self, job_resources):  # pragma: no cover
```

Comment:
This may be premature, but after looking at Paul's PR #278. I'm wondering how those changes will affect this ticket specifically around the language of job card vs. batch script, and the movement of the code.

Reply:
Definitely not premature! Pending that resolution it may have to change

Reply:
It might make for a messy merge. I will happily volunteer a follow-on PR targeting _this branch_ (and its PR) that resolves any conflicts and preserves the in-progress work on this branch.

Reply:
Haha oh I've been keeping an eye on that one, don't worry!

Reply:
Well, the answer is "batch script"! On that update and will wrangle the subsequent resolutions :)